doc/userguide/output/lua-output.rst

```diff
@@ -1,17 +1,25 @@
 Lua Output
 ==========
 
-Lua scripts can be used to generate output from Suricata.
+Suricata offers the possibility to get more detailed output on specific kinds of
+network traffic via pluggable lua scripts. You can write these scripts yourself and only need to
+define four hooking functions.
```

Comment:
I would write *hook functions* :)

Reply:
Ahh okay, just looked it up, the common term really is *hook functions*, no idea how I came to hooking functions